we don't need to unlock it after erroring, because the parent function either
unlocks it for us, or it fails horribly so we don't care about the mutex state

diff --git a/vendor/code.rocketnine.space/tslocum/cview/application.go b/vendor/code.rocketnine.space/tslocum/cview/application.go
index de2cec5..b46141a 100644
--- a/vendor/code.rocketnine.space/tslocum/cview/application.go
+++ b/vendor/code.rocketnine.space/tslocum/cview/application.go
@@ -234,11 +234,9 @@ func (a *Application) init() error {
 	var err error
 	a.screen, err = tcell.NewScreen()
 	if err != nil {
-		a.Unlock()
 		return err
 	}
 	if err = a.screen.Init(); err != nil {
-		a.Unlock()
 		return err
 	}
 	a.width, a.height = a.screen.Size()
